### PR TITLE
[v1.0.0] update docs

### PR DESCRIPTION
**Pull request by gritzkoo**

### CODE REVIEW

**Review by dioculos — COMMENTED** [review]
Looks good to me!

File: README.md [hunk 1]
```diff
@@ -14,75 +14,24 @@
...
-  }).catch( error => {
-    check.error = error
-    resolve(check)
-  })
-}
-```
+A successor package for [nodejs-health-checker](https://github.com/gritzkoo/nodejs-health-checker) to simplify health checks.
 
-## How to init
+The main purpose of this package is to substitute `nodejs-health-checker` package and standardize the liveness and readiness actions for Nodejs applications running in Kubernetes deployments, without adding complexity and extra package installs.
```
Comment (joaomantovani) [inline README.md:26]: Add the link of the old:

https://github.com/gritzkoo/nodejs-health-checker

Comment (joaomantovani) [reply README.md:26]:
```suggestion
The main purpose of this package is to substitute [`nodejs-health-checker`](https://github.com/gritzkoo/nodejs-health-checker) package and standardize the liveness and readiness actions for Nodejs applications running in Kubernetes deployments, without adding complexity and extra package installs.
```

File: README.md [hunk 2]
```diff
@@ -14,75 +14,24 @@
...
-      // name is just a string that will help you fast identify the integration you need to check-on
-      name: 'github integration',
-      //
-      handle: MyIntegrationTest
-      }
-    }
-  ]
-})
-```
+Read more about migrating from `nodejs-health-checker` or creating your tests in [MIGRATIONS GUIDELINES](docs/MIGRATIONS.md)
```
Comment (joaomantovani) [inline README.md:28]: Same here: [`nodejs-health-checker`](https://github.com/gritzkoo/nodejs-health-checker)

Comment (joaomantovani) [reply README.md:28]:
```suggestion
Read more about migrating from [`nodejs-health-checker`](https://github.com/gritzkoo/nodejs-health-checker) or creating your tests in [MIGRATIONS GUIDELINES](docs/MIGRATIONS.md)
```

**Comment by joaomantovani** [comment]
Just a few improvements! The PR is good to merge. 👍

**Pull request merged**